File: app/src/main/java/com/asmat/rolando/popularmovies/ui/adapters/FavoriteMoviesGridAdapter.java
package com.asmat.rolando.popularmovies.ui.adapters;

import com.asmat.rolando.popularmovies.R;

public class FavoriteMoviesGridAdapter extends MoviesGridBaseAdapter {
    public FavoriteMoviesGridAdapter(MovieAdapterOnClickHandler handler) {
        super(handler);
    }

    @Override
    public int getEmptyStateLayoutID() {
        return R.layout.empty_state_favorites;
    }
}

File: app/src/main/java/com/asmat/rolando/popularmovies/ui/mylists/FavoriteMoviesGridAdapter.kt
package com.asmat.rolando.popularmovies.ui.mylists

import com.asmat.rolando.popularmovies.R
import com.asmat.rolando.popularmovies.ui.common.MovieAdapterOnClickHandler
import com.asmat.rolando.popularmovies.ui.common.BaseMoviesGridAdapter

class FavoriteMoviesGridAdapter(handler: MovieAdapterOnClickHandler) : BaseMoviesGridAdapter(handler) {

    override val emptyStateLayoutID: Int
        get() = R.layout.empty_state_favorites

}